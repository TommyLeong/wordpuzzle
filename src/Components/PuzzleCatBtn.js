import React from 'react';
import {View, Text, TouchableOpacity} from 'react-native';

const PuzzleCatBtn = props => {
  return (
    <View>
      <TouchableOpacity
        style={{
          height: 70,
          margin: 15,
          backgroundColor: 'pink',
          borderRadius: 15,
          justifyContent: 'center',
          alignItems: 'center',
        }}
        onPress={() => {
          props.navigation.navigate('PuzzleScreen', {category: props.catName});
        }}>
        <Text>{props.catName}</Text>
      </TouchableOpacity>
    </View>
  );
};

export default PuzzleCatBtn;
