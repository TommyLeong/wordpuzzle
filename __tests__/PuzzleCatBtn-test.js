import React from 'react';
import renderer from 'react-test-renderer';
import PuzzleCatBtn from '../src/Components/PuzzleCatBtn';

test('renders correctly', () => {
  const tree = renderer.create(<PuzzleCatBtn />).toJSON();
  expect(tree).toMatchSnapshot();
});
