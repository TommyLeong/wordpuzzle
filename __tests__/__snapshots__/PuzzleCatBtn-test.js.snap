// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "pink",
        "borderRadius": 15,
        "height": 70,
        "justifyContent": "center",
        "margin": 15,
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
</View>
`;
